### DISCUSSION TITLE
DatePicker 选中的日期背景白色bug

### DISCUSSION THREAD

**Discussion by sxbjxyy** [dotnetcore/BootstrapBlazor#3329] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

DatePicker 选中的日期背景白色bug，官网也是如此

### Expected Behavior

DatePicker 选中的日期背景与未选中不同，给加个背景色即可

**Comment by bb-auto** [comment]
@sxbjxyy Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by ArgoZhang** [comment, marked as answer]
@sxbjxyy 我这里一起正常，建议 `F12` 后清除缓存试一下

请点击👇🏻 `Mark as answer`
